app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    data = {}
+
+    def wrapper(*args) -> None:
+        if args in data:
```

Comment:
Using mutable types as dictionary keys can lead to issues since they are not hashable. Ensure that `args` are hashable (e.g., tuples of immutable types) when used as keys in the `data` dictionary.